webpack/assets/javascripts/bundle.js

```diff
@@ -8,7 +8,13 @@ require('expose?JsDiff!diff');
 require('./bundle_flot');
 require('./bundle_select2');
 require('./bundle_datatables');
-window.tfm = {
-  tools: require('./foreman_tools'),
-  numFields: require('./jquery.ui.custom_spinners')
-};
+
+window.tfm = Object.assign(
```

Comment:
```js 
window.tfm = window.tfm || {}; 
```
is preferred over 
```js 
window.tfm = { ...}
```
Otherwise important objects could be easily overridden.

Originally I wrote it as follows:
```js
window.tfm = window.tfm || {};
window.tfm.tools = require('./foreman_tools');
window.tfn.numFields: require('./jquery.ui.custom_spinners'),
window.tfm.reactMounter = require('./react_app/common/MountingService');
```
Tomer prefers the object syntax so this is a sort of compromise